tutorials/assets_pipeline/importing_scenes.rst

```diff
@@ -29,6 +29,33 @@ Copy the scene file together with the textures and mesh data (if separate) to
 the project repository, then Godot will do a full import when focusing the
 editor window.
 
+3D asset direction conventions
+------------------------------
+
+Godot uses a right-handed, Y-is-up coordinate system, with the -Z axis as
+the camera's forward direction. This is the same as OpenGL. This implies
+that +Z is back, +X is right, and -X is left for a camera.
+
+The convention for 3D assets is to face the opposite direction as the camera,
+so that characters and other assets are facing the camera by default.
```

Comment:
This part is confusing me (and making me realize my 3D teachers in college taught even less than I thought, which wasn't much). What exactly is "the camera" that you're referring to here? I assume it's not one you would attach to a player character

Reply:
@skyace65 Does this image help you understand?

![image](https://user-images.githubusercontent.com/6265307/232059168-bd9d9dcf-b06d-459c-9267-1b7ecb7d4931.png)

Reply:
It may be paraphrased as a viewpoint of the viewport.

Reply:
So the camera in this case would be the viewport camera when you a open a blank scene?

Reply:
In most cases, yes.